_commands/basics/zip.md

```diff
@@ -0,0 +1,36 @@
+---
+---
+
+zip
+---
+
+'zip' is used to compress multiple files* into a .zip file
+or to compress a directory^ into a .zip file.
+
+~~~ bash
+*zip combined.zip file1 file2 file3
```

Comment:
what are '*' and '^' being used to show. If you're trying to show a line of bash, start it with a '$'